Convert CI from Travis to GitHub Actions

krievley opened the pull request:
Added the yml file to run the travis process on github actions.

mattstauffer commented on .github/workflows/run-php-tests.yml:
```diff
@@ -0,0 +1,41 @@
+
+on:
+  push:
+    branches: [ krc/add-github-actions ]
```
You'll want to update this to `main` before we merge

mattstauffer commented on .github/workflows/run-php-tests.yml:
```diff
@@ -0,0 +1,41 @@
+      run: composer self-update
+    
+    - name: Composer Update
+      run: composer update ${{ matrix.composer_flag_string }} --no-interaction --prefer-source
```
It looks like this isn't actually running the tests?

mattstauffer commented:
Please delete any Travis files while you're at it, and add the GH Actions badge to the readme

krievley commented:
Matt, per your feedback I have tried adding better commit messages and performed the updates you requested:

1. Changed the branch to watch to "main".
2. Added a PHPUnit step to the config.
3. Removed the Travis CI file.
4. Added GitHub Actions Badge to the README

mattstauffer commented:
Thanks! One remaining note on commits: Please use the present imperative tense, so "Add" instead of "Added", "Create" instead of "Created", etc.

Additionally, you left the Travis badge still in the readme.

Please take the time to read each PR you make before submitting it so you can see if there's anything you could potentially catch before it makes it to the reviewer. Thanks!

mattstauffer commented:
Thanks @krievley!!
